js/interface.js

```diff
@@ -236,19 +236,27 @@ var DynamicLists = (function() {
           var type = $(this).data('field');
           var $selector = $(this).parents('.filter-panel');
 
-          if (type === 'field') {
-            $selector.find('.panel-title-text .column').html(value === 'none' ? '(Field)' : value);
-          }
+          switch (type) {
+            case 'field':
+              $selector.find('.panel-title-text .column').html(value === 'none' ? '(Field)' : value);
+              break;
 
-          if (type === 'logic') {
-            var hideValueFields = value === 'empty' || value === 'notempty';
+            case 'logic':
```

Comment:
what does "case logic" mean?

Reply:
@squallstar 
This logic works when changing any dropdown in the filter list, where type is responsible for which dropdown was changed.
![изображение](https://user-images.githubusercontent.com/71251809/103281680-25c17400-49dc-11eb-8d41-852b69aaa436.png)

Reply:
@tonytlwu can you confirm the screenshot above is in line with your expectations both in terms of UI and naming of the variables?

It's a little confusing to me (without knowing the feature I would have expected `type` to have values from a dropdown rather than be hardcoded from labels/underlying logic)

Reply:
@squallstar 
Such an implementation has already been added earlier, I decided to change the structure instead of if conditions, I changed to use a switch case as there was a need to add a new condition

Reply:
I see, that makes sense thanks for explaining.